gateway_to_backend/protocol_buffers_files/otap_message.proto

```diff
@@ -24,11 +24,23 @@ message GetScratchpadStatusResp {
 }
 
 message UploadScratchpadReq {
+    message ChunkInfo {
+        // Id of the scratchpad this chunk belongs to
+        required uint32 scratchpad_unique_id = 1;
```

Comment:
After thinking it twice, this field seems a bit useless. There is already the seq on the outer message.
It may create more confusion than it may help.
It is up to the sender to ensure that chunks are from the same full scratchpad, and this id doesn't really help.